bridgetown-core/lib/site_template/Gemfile.erb

```diff
@@ -17,3 +17,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Happy Bridgetowning!
 
 gem "bridgetown", "~> <%= Bridgetown::VERSION %>"
+
+# Puma is a Rack-compatible server
+# (you can optionally limit this to the "development" group)
+gem "puma", "~> 5.2"
```

Comment:
Maybe it's worth putting this in `development` by default? I'd imagine the vast majority of people would use it only use it locally.

Reply:
I guess the question is which DX is worse: installing the puma gem in production even if it's not required for a static site build, or _not_ installing it even when someone might need to utilize SSR? I tend to think the latter would be annoying and we'd have to document it: "hey, wanna do SSR? you need to move puma out of development in the Gemfile…" hmm

Reply:
Yeah fair point. It's not doing any harm outside the dev group and saves the need to go and look up some docs 👍